<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createActivity</class>
 <widget class="QDialog" name="createActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>*Activity Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>*Activity Type :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTestID">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_activityType">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Essay</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Problem Set</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Programming Assignment</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Multiple Choice</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>For Programming Activtity Only</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>*Path to Solutions :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_createActivity">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Activity</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dueDate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Number Of Input Files: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Test:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Path to Student Submission Directory :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SearchSol">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtActivityName">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtStdDir">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>309</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SearchSub">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>200</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Number Of Output Files:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_numTests">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>120</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>653</x>
     <y>10</y>
     <width>20</width>
     <height>421</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_activityLang">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>English</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>French</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>German</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chinese</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C++</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Java</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Python</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="Numtests">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="numOutput">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>200</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_environment">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>240</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>c++</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Java</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Eclipse</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QT</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txtSolutionPath">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>299</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Due Date :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Test :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numInput">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>170</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>240</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Environment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>*Activity Language :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_testID">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*testID:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
